frontend/src/stores/settings.ts

```diff
@@ -24,8 +24,9 @@ export const useSettingsStore = defineStore({
     "audio.in.squelch": 3,
     "led.max_brightness": 5,
     "beacon.mode": BeaconMode.MORSE_CODE,
-    "beacon.text": "-..--.",
+    "beacon.text": "Default text",
     "beacon.delay_seconds": 12,
+    "beacon.morse_code.text": "-.. . ..-. .- ..- .-.. - / - . -..- -",
```

Comment:
`beacon.morse_code.text` should not be in the store as we need to have single source of truth. Instead in the Vue component you can use `watch` and calculate the morse code value dynamically based on the value of `beacon.text`.